tests/test_job_management.py

```diff
@@ -27,12 +27,31 @@ def test_callback_without_prior_run_analysis(dispatcher_live_fixture):
 
     assert c.status_code == 200
 
-#@app.route('/call_back', methods=['POST', 'GET'])
-#def dataserver_call_back():
-    #log = logging.getLogger('werkzeug')
-    #log.disabled = True
-    #app.logger.disabled = True
-#    print('===========================> dataserver_call_back')
-#    query = InstrumentQueryBackEnd(
-#        app, instrument_name='mock', data_server_call_back=True)
-#    query.run_call_back()
\ No newline at end of file
+
+def test_callback_after_run_analysis(dispatcher_live_fixture):
+    server = dispatcher_live_fixture
+    print("constructed server:", server)
+
+    c = requests.get(server + "/run_analysis",
+                     params=dict(
+                        query_status="new",
+                        query_type="Real",
+                        instrument="empty",
+                        product_type="numerical",                        
+                        async_dispatcher=False,
+                    ))
+
+    print(c.text)    
+
+    assert c.status_code == 200        
+
+    c = requests.get(server + "/call_back",
+                   params={
+                       'job_id': 'test-job-id',
```

Comment:
Is this job-id completely randomly choosen or does it have a purpose?

Reply:
no purpose in this case. As we advance the job management testing, it will correspond to a directory which must exist.
Right now it just covers a case  which otherwise leads to an unhandled exception.